capstone/src/main/java/com/google/step/search/RestaurantQueryParams.java

```diff
@@ -0,0 +1,81 @@
+package com.google.step.search;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Class to hold data for restaurant search queries.
+ */
+public class RestaurantQueryParams {
+  private final String query;
+  private final List<String> cuisine;
+  private final String email;
+
+  public static class Builder {
+    private String query = "";
+    private List<String> cuisine = new ArrayList<>();
+    private String email = "";
+
+    public Builder() { }
+
+    /**
+     * Sets query parameter to {@code query}
+     * @param query String representing a search query
+     * @return {@code this} builder
+     */
+    public Builder setQuery(String query) {
```

Comment:
With builder patterns, it's actually best not to make these look like 'setters'. Just call this method `query`, then when you invoke the builder, it'll look cleaner (e.g. `new RestaurantQueryParams.Builder().query("foo").build()`).

Reply:
Changed this. Thanks for the tip!